dff/stats/__init__.py

```diff
@@ -1,2 +1,9 @@
 # -*- coding: utf-8 -*-
 # flake8: noqa: F401
+
+from .utils import get_wrapper_field, STATS_KEY
```

Comment:
We can remove `STATS_KEY`, we don't use it, that's right?

Reply:
Correct